notebooks/Emukit-basic-use-of-the-library.ipynb

```diff
@@ -0,0 +1,349 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Basic use of the Emukit\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# General imports and parameters of figures should be loaded at the beginning of the overview\n",
+    "\n",
+    "### General imports\n",
+    "%matplotlib inline\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import colors as mcolors\n",
+    "\n",
+    "### --- Figure config\n",
+    "colors = dict(mcolors.BASE_COLORS, **mcolors.CSS4_COLORS)\n",
+    "LEGEND_SIZE = 15\n",
+    "TITLE_SIZE = 25\n",
+    "AXIS_SIZE = 15"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The goal of this notebook is to illustrate how the a model can wrapped and used in different tasks in Emukit."
```

Comment:
"...how _the_ _a_ model can wrapped.." -> "how _a_ model can _be_ wrapped"

Reply:
This typo is still there
```suggestion
    "The goal of this notebook is to illustrate how a model can wrapped and used in different tasks in Emukit."
```